components/camel-olingo4/camel-olingo4-api/src/test/java/org/apache/camel/component/olingo4/Olingo4AppAPITest.java

```diff
@@ -405,6 +414,70 @@ public void testBatchRequest() throws Exception {
         assertEquals(HttpStatusCode.NOT_FOUND.getStatusCode(), responseParts.get(7).getStatusCode());
     }
 
+    @Test
+    public void testUnboundActionRequest() throws Exception {
+        final TestOlingo4ResponseHandler<HttpStatusCode> responseHandler = new TestOlingo4ResponseHandler<>();
+        olingoApp.action(edm, TEST_UNBOUND_ACTION_RESETDATASOURCE, null, null, responseHandler);
+
+        final HttpStatusCode statusCode = responseHandler.await(15, TimeUnit.MINUTES);
```

Comment:
Is there really needed await for 15 minutes? I dont feel comfortable waiting 15 minutest for test to fail.

Reply:
No, no reason. Just a copy & paste from batch test issue. Fixed now.